src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -36,7 +37,7 @@ protected function doClone($var)
         $maxItems = $this->maxItems;
         $maxString = $this->maxString;
         $cookie = (object) array();     // Unique object used to detect hard references
-        $gid = uniqid(mt_rand(), true); // Unique string used to detect the special $GLOBALS variable
+        $gid = \spl_object_hash($cookie); // Unique string used to detect the special $GLOBALS variable
```

Comment:
Not sure if that's relevant in this context, but `spl_object_hash` does not guarantee that it returns a unique value. The value can  be reused when objects are reclaimed. And we did hit that limitation in the past (and Doctrine as well), so that's not just theoretical.